Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Borgs/borgs_tools.yml

```diff
@@ -59,10 +61,11 @@
           Quantity: 1
 
 - type: entity
-  name: medical robot health analyzer
   parent: BaseItem
-  id: BorgHealthAnalyzer
-  description: A hand-held body scanner capable of distinguishing vital signs of the subject.
+  id: HandheldHealthAnalyzerBorg
+  name: health analyzer
+  suffix: Robot, No Map
+  description: A batteryless hand-held body scanner meant for medical robots capable of distinguishing vital signs of the subject.
```

Comment:
Better more immersive description would say "runs off of your own power cells" instead of "batteryless"